graphdatascience/pipeline/pipeline_endpoints.py

```diff
@@ -13,16 +13,44 @@ def pipeline(self) -> PipelineProcRunner:
         return PipelineProcRunner(self._query_runner, f"{self._namespace}.pipeline", self._server_version)
 
     def lp_pipe(self, name: str) -> LPTrainingPipeline:
```

Comment:
same as above these are not used in the generated docs?

Reply:
These ones are actually used. Because they are in PipelineEndpoints <- DirecteEndpoints <- GraphDataScience.
All docstrings if they are there should be used, proc runners should not have docstrings and I'll remove those leftover.